k8s/templates/frontend-pgbouncer.yaml

```diff
@@ -0,0 +1,99 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "studio.fullname" . }}-dbpool-config
+  labels:
+    app: {{ template "studio.fullname" . }}
+data:
+  DB_HOST: {{ .Values.postgresql.externalCloudSQL.proxyHostName | default (include "postgresql.fullname" .) }}
+  # Aron: I configured pgbouncer based on the settings
+  # recommended by this article:
+  # https://medium.com/futuretech-industries/postgres-raffle-10k-connections-35-mo-part-two-b4c2e0c86e37
+  POOL_MODE: "transaction"
+  SERVER_RESET_QUERY: "DISCARD ALL"
+  # the custom listen backlog doesn't matter until we increase the nodes' somaxconn
+  LISTEN_BACKLOG: "16384"
```

Comment:
when we change the node's somaxconn later, will the value of this variable still be 16384? just wondering whether this value can be super large without affecting other resources of the server. thank you!

Reply:
Yup! It'll increase the size of some kernel data structures, but that shouldn't be an issue since the impact is still minimal overall.

Reply:
sounds good!